app/controllers/feedback_messages_controller.rb

```diff
@@ -31,7 +31,9 @@ def create
   private
 
   def recaptcha_verified?
-    recaptcha_params = { secret_key: ApplicationConfig["RECAPTCHA_SECRET"] }
+    return true if SiteConfig.recaptcha_site_key.blank? && SiteConfig.recaptcha_secret_key.blank?
```

Comment:
@Zhao-Andy perhaps its late and i'm tired but isn't recaptcha_verified? supposed to return false if the SiteConfig.recaptcha_site_key or secret key is blank?

Reply:
All good :)

I want to return true because if they're blank, we're assuming they don't want to use / haven't set recaptcha. This allows them to submit abuse reports without recaptcha.

I was thinking maybe this should be another SiteConfig variable that's just a boolean, like `SiteConfig.recaptcha_enabled` but that seemed unnecessary.

Reply:
**(non-blocking, suggestion)**: hmm I see what you mean. I think I was confused by the method name vs. what we doing in that specific line of code. When we return true from that line, it's not because it was verified but rather because we don't want to use recaptcha which makes the naming of the function confusing. 

Perhaps there's some way to make this easier to read, like extracting that condition into its own descriptive function.

Reply:
Updated!